lineman/app/app.coffee

```diff
@@ -47,6 +47,7 @@ angular.module('loomioApp').controller 'AppController', ($scope, $filter, $rootS
     {path: '/inbox', component: 'inboxPage' },
     {path: '/groups', component: 'groupsPage' },
     {path: '/profile', component: 'profilePage'},
+    {path: '/email_settings', component: 'emailSettingsPage' },
```

Comment:
Hey it feels appropriate to nest this under `/profile` I think, could we do either `profile/settings` or `profile/email_settings`?